modules/CmpdRegAdmin/src/client/AbstractCmpdRegAdminBrowser.coffee

```diff
@@ -173,9 +174,9 @@ class window.AbstractCmpdRegAdminBrowserController extends Backbone.View
 			urlRoot: "/api/CmpdRegAdmin/#{@entityTypePlural}"
 			toDisplay: @toDisplay
 		@searchController.render()
-		@searchController.on "searchRequested", @handleSearchRequested
-		@searchController.on "searchReturned", @setupCmpdRegAdminSummaryTable
-		@searchController.on "createNewCmpdRegAdmin", @handleCreateNewCmpdRegAdminClicked
+		@searchController.on "searchRequested", @handleSearchRequested.bind(@)
```

Comment:
listening to `on` triggers requires that you bind this.  we got away with this because of the way coffeescript 1 compiled our classes but not with coffeescript2 and the window. change.